src/_baseMenu.js

```diff
@@ -889,30 +888,24 @@ class BaseMenu {
   /**
    * Handles click events throughout the menu for proper use.
    *
-   * Depending on what is supported either `touchstart` and `touchend` or
-   * `mousedown` and `mouseup` will be used for all "click" event handling.
+   * `pointerdown` and `pointerup` will be used for all "click" event handling.
    *
-   * - Adds a `touchend`/`mouseup` listener to the document so if the user clicks
+   * - Adds a `pointerup` listener to the document so if the user clicks
    *   outside of the menu when it is open, the menu will close.
-   * - Adds a `touchstart`/`mousedown` listener to every menu item that will blur
+   * - Adds a `pointerdown` listener to every menu item that will blur
    *   all menu items in the entire menu structure (starting at the root menu) and
    *   then properly focus the clicked item.
-   * - Adds a `touchend`/`mouseup` listener to every submenu item that will properly
+   * - Adds a `pointerup` listener to every submenu item that will properly
    *   toggle the submenu open/closed.
-   * - Adds a `touchend`/`mouseup` listener to the menu's controller
+   * - Adds a `pointerup` listener to the menu's controller
    *   (if the menu is the root menu) so when it is clicked it will properly
    *   toggle open/closed.
    *
    * @protected
    */
   _handleClick() {
-    // Use touch over mouse events when supported.
-    const startEventType = isEventSupported("touchstart", this.dom.menu)
-      ? "touchstart"
-      : "mousedown";
-    const endEventType = isEventSupported("touchend", this.dom.menu)
-      ? "touchend"
-      : "mouseup";
+    const startEventType = "pointerdown";
+    const endEventType = "pointerup";
```

Comment:
Since there's no longer any logic behind these we'll just use the strings in the events themselves.

```suggestion
```